src/Utilities/VulnerabilityRoutes.js

```diff
@@ -54,17 +54,12 @@ export const checkForAccountSystems = (isEdgeParityEnabled, hasEdgeDevices, hasC
     return isEdgeParityEnabled && (hasEdgeDevices || hasConventionalSystems) || hasConventionalSystems;
 };
 
-export const InsightsElement = ({ element: Element, title, globalFilterEnabled, ...elementProps }) => {
-    let location = useLocation();
-    const [isLoading, setLoading] = useState(true);
-    const [hasConventionalSystems, setHasConventionalSystems] = useState(true);
+export const useAccountStats = (isEdgeParityEnabled, setLoading, addNotification) => {
```

Comment:
Just to clean up the file a bit, let's move this to a separate file other than this.